lib/gems/pending/appliance_console/internal_database_configuration.rb

```diff
@@ -82,6 +83,7 @@ def initialize_postgresql_disk
 
     def initialize_postgresql
       log_and_feedback(__method__) do
+        check_pg_mount_point
```

Comment:
I think this would make more sense as a check in `activate` or even in ask_questions.

Right after choose disk we could print some error message `unless disk || have_pg_mount_point?`